server/app/models/user.js

```diff
@@ -21,7 +21,7 @@ var UserSchema = new Schema({
   },
   password: {
     type: String,
-    required: 'Password is required',
+  //  required: 'Password is required',   //has to be optional due facebook etc.,  autho password is not require 
```

Comment:
:eyes: Nope, password should always be required.
When user signing up through _Facebook_ or _Google+_ they should also add password by themselves.